order_delivery_date.php

```diff
@@ -699,6 +700,19 @@ public static function orddd_lite_admin_rate_us( $footer_text ) {
 				return $footer_text;
 			}
 		}
+		/**
+		 * Bakery_enqueue_scripts.
+		 *
+		 * @since 3.11.0
+		 * @param array $scripts The all scripts name.
+		 * @return array
+		 */
+		public function bakery_enqueue_scripts( $scripts ) {
+			if ( 'bootstrap-datepicker' === $scripts[5] ) {
+				unset( $scripts[5] );
```

Comment:
can we add this check in foreach loop so that there will be no issues if $scripts values are changed?